src/ts/components/seekbar.ts

```diff
@@ -763,6 +763,11 @@ export class SeekBar extends Component<SeekBarConfig> {
       this.setSeeking(false);
       seeking = false;
 
+      // Update the UI in case we only have a click or touch
+      this.setSeekPosition(targetPercentage);
+      this.setPlaybackPosition(targetPercentage);
+
+
```

Comment:
```suggestion
      // Update the UI in case we only have a click or touch
      this.setSeekPosition(targetPercentage);
      this.setPlaybackPosition(targetPercentage);

```

too generous, dude 😄